simple-chat-front/src/components/Chat/Chat-Style.css

```diff
@@ -0,0 +1,70 @@
+.chat{
+    width: 70%;
+    height: 75% ; 
+    display: flex;
+    flex-direction: column;
+}
+
+.chat__header{
```

Comment:
the usual convention for CSS that I've seen is dashes not underscored but if that works no problem

Reply:
The convention we're using is BEM, check it out if you want to http://getbem.com/naming/

Reply:
awesome, I'll do it, thanks